GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+    The vote is deemed successful if at least two thirds of voters express a lack of confidence.
+    * If this is a single-member vote called by Council Members, only Council Members vote, and the 
+    vote is deemed successful if at least half the voters express a lack of confidence. 
+    Council Members also have the possibility to call for the whole core team to vote 
+    (i.e Council members and Core contributors), although this is not the default option. 
+    The threshold for successful vote is also at 50% of voters for this option.
+* If a single-member vote succeeds, then that member is removed from the council and the resulting 
+vacancy can be handled in the usual way.
+* If a whole-council vote succeeds, the council is dissolved and a new council election is triggered
+immediately.
+
+#### Ejecting Core Contributors
+
+* Core contributors can be ejected through a simple majority vote by the council. 
+Council members vote "Yes" or "No". 
+* Upon ejecting a core contributor the council must publish an issue ticket, or public document 
+detailing the
+    * Violations
+    * Evidence if available
+    * Remediation plan (if necessary)
+    * Signatures majority of council members to validate correctness and accuracy
+
+#### Leaving the project
+
+Core contributors can also voluntarily leave the project by notifying the community through a public
+means or by notifying the entire council.
+
+Unless they request otherwise, they will be listed on {ref}`emeritus` page. <!-- TODO ref -->
+
+#### Voting Criteria For Future Elections
+
+Voting for first election is restricted to establish stable governance, and to defer major decision to elected leaders
+* For the first election only the people registered following the guidelines in `elections/ArviZ_2020.md` can vote
+* In the first year, the council must determine voting eligibility for future elections between two criteria:
+  * Core contributors
+  * The contributing community at large
+
+## Historical note
+
+Traditionally, project leadership was unstructured but primarily driven by a subset of 
+Core Contributors whose active and consistent contributions have been recognized by their receiving 
+"commit rights" to the Project GitHub repositories. 
+In general all Project decisions are made through consensus among the Core Contributors with input 
+from the Community.
+
+While this approach has served us well, as the Project grows and faces more legal and financial 
+decisions and interacts with other institutions, we see a need for a more formal governance model. 
+Moving forward The Project leadership will consist of a **TODO NAME** Council. 
+We view this governance model as the formalization of what we are already doing, rather than a 
+change in direction.
+
+
+<!-- NOTES:  
+
+Include something about our relationship with PyMC (PyMC helped us by giving us access to GSoC three times)
+How do we start a Council if we don't have 'core' contributors yet?
```

Comment:
Identify core contributors first through some critieria. Examples are asking anyone that's contributed to self nominate, that way you can start with folks who are involved and actively interested.